examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+"""
+
+.. _metas_cbmr:
+
+===========================================
+Coordinate-based meta-regression algorithms
+===========================================
+
+A tour of CBMR algorithms in NiMARE
+
+This tutorial is intended to provide a brief description and example of the CBMR
+algorithm implemented in NiMARE. For a more detailed introduction to the elements
+of a coordinate-based meta-regression, see other stuff.
+"""
+import numpy as np
+import scipy
+from nilearn.plotting import plot_stat_map
+
+from nimare.generate import create_coordinate_dataset
+from nimare.meta import models
+from nimare.transforms import StandardizeField
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+
+# data simulation
+ground_truth_foci, dset = create_coordinate_dataset(foci=10, sample_size=(20, 40), n_studies=1000)
+# set up group columns: diagnosis & drug_status
+n_rows = dset.annotations.shape[0]
+dset.annotations["diagnosis"] = [
+    "schizophrenia" if i % 2 == 0 else "depression" for i in range(n_rows)
+]
+dset.annotations["drug_status"] = ["Yes" if i % 2 == 0 else "No" for i in range(n_rows)]
+dset.annotations["drug_status"] = (
+    dset.annotations["drug_status"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+# set up continuous moderators: sample sizes & avg_age
+dset.annotations["sample_sizes"] = [dset.metadata.sample_sizes[i][0] for i in range(n_rows)]
+dset.annotations["avg_age"] = np.arange(n_rows)
+# set up categorical moderators: schizophrenia_subtype (as not enough data to be interpreted
+# as groups)
+dset.annotations["schizophrenia_subtype"] = ["type1", "type2", "type3", "type4", "type5"] * int(
+    n_rows / 5
+)
+dset.annotations["schizophrenia_subtype"] = (
+    dset.annotations["schizophrenia_subtype"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+
+###############################################################################
+# Estimation of group-specific spatial intensity functions
```

Comment:
This is nice. I would move this up higher up in the notebook (where I said you should have a high level overview